client/my-sites/stats/feedback/index.tsx

```diff
@@ -214,7 +217,11 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 	return (
 		<div className="stats-feedback-container">
 			<FeedbackCard clickHandler={ handleButtonClick } />
-			<FeedbackPanel isOpen={ isFloatingPanelOpen } clickHandler={ handleButtonClick } />
+			<FeedbackPanel
+				isOpen={ isFloatingPanelOpen }
+				animationClassName={ animationName }
```

Comment:
This is similar to my comment below about the animation and the visibility not being 1-to-1. I see what you mean but I'm not sure the best approach here. If we want to manage the animation internally, I think we'd need to tweak it to include a callback so that the caller knows when the animation has completed.

Reply:
I guess it's a solvable problem with my suggestion above. Yeah, there are events we could potentially listen to some [events](https://developer.mozilla.org/en-US/docs/Web/API/Element/animationend_event) so that we know precisely, but I'm not sure whether it's worth it.

Reply:
> animationClassName is an implementation detail we probably want to hide from the caller

I agree with this. That may be too coupling.